House-of-glass/src/components/Filters/FilterCompose.jsx

```diff
@@ -0,0 +1,56 @@
+
+
+const hightolow =(state,array)=>{
+    let sortedList = state.sortBy==="HIGHTOLOW"?[...array].sort((a,b)=>Number(b.price)- Number(a.price)):[...array];
+    return sortedList
+}
+
+const lowtohigh =(state,array)=>{
+    let sortedList = state.sortBy==="LOWTOHIGH"?[...array].sort((a,b)=>Number(a.price)- Number(b.price)):[...array]
+    return sortedList
+}
```

Comment:
This can be a single function with a switch case for "HIGHTOLOW" and "LOWTOHIGH"

Reply:
Yes, I've implemented a single function to tackle both the sort types. Thank you for your suggestion.